### DISCUSSION TITLE
Trying to create a new event from frontmatter with meets conferencing

### DISCUSSION THREAD

**Discussion by delbirt-cpr** [YukiGasai/obsidian-google-calendar#137] [General]
Working to create events from frontmatter and running into a funny issue where my Google Meets links don't appear.

Following the API documentation, [here](https://developers.google.com/calendar/api/v3/reference/events/insert), and some example code, [here](https://gist.github.com/tanaikech/94791d48823e9659aa376cf7f0161d9b), I'm trying to figure out where I can add the request parameter "conferenceDataVersion=1".

The following _should work_ provided I can get that parameter added to the request.

```
{
	"kind":"calendar#event",
	"summary": "My fancy event!",
	"description": "-Sent from my Obsidian vault :D",
	"start": {
		"dateTime": "2023-06-20T13:50:00-0400"
		},
	"end": {
		"dateTime": "2023-06-20T14:00:00-0400"
		},
	"attendees": [
		{
			"email": "email@organizer.com",
			"organizer": "True"
		},
		{
			"email": "someotherguy@colleague.com"
		}
	],
	"conferenceData": {
		"createRequest": {
			"requestID": "083unjfsld",
			"conferenceSolutionKey": {
				"type": "hangoutsMeet"
			}
		}
	}
}
```

**Comment by YukiGasai** [comment]
Hi @delbirt-cpr,
I moved this to an actual issue and fixed this with a new version.
Please check out #138.
